zebra-network/src/peer_set/set.rs

```diff
@@ -410,6 +418,26 @@ where
         }
     }
 
+    /// Checks if the minimum peer version has changed, and disconnects from outdated peers.
```

Comment:
Nit: This method does not disconnect from unready peers.

```suggestion
    /// Checks if the minimum peer version has changed, and disconnects from outdated ready peers.
```